python/fbprophet/hdays.py

```diff
@@ -1430,3 +1432,62 @@ def _populate(self, year):
 
 class KR(Korea):
     pass
+
+
+# ------------ Holidays in Belarus---------------------
+class Belarus(HolidayBase):
+    """
+    Implement public holidays in Belarus
+
+    Reference:
+    https://en.wikipedia.org/wiki/Public_holidays_in_Belarus
+
+    Please note:
+    Some holidays might collide with weekends and therefore not compensated with next business day
+    as International Women's Day
+    """
+
+    def __init__(self, **kwargs):
+        self.country = "BY"
+        HolidayBase.__init__(self, **kwargs)
+
+    def _populate(self, year):
+        # New Year's Day
+        name = "New Year's Day"
+        self[date(year, 1, 1)] = name
+
+        # Orthodox Christmas day
+        name = "Orthodox Christmas Day"
+        self[date(year, 1, 7)] = name
+
+        # International Women's Day
+        name = "International Women's Day"
+        self[date(year, 3, 8)] = name
+
+        # Commemoration Day
+        name = "Commemoration Day"
+        self[easter(year, EASTER_ORTHODOX) + timedelta(days=9)] = name
+
+        # Spring and Labour Day
+        name = "Spring and Labour Day"
+        self[date(year, 5, 1)] = name
+
+        # Victory Day
+        name = "Victory Day"
+        self[date(year, 5, 9)] = name
+
+        # Independence Day
+        name = "Independence Day"
+        self[date(year, 6, 3)] = name
```

Comment:
It looks like this should be July?

Reply:
Sorry, fixed + hardcode replaced with an imported variable